cobalt/media/base/sbplayer_bridge.cc

```diff
@@ -731,6 +731,13 @@ void SbPlayerBridge::CreatePlayer() {
 
   is_creating_player_ = true;
 
+  if (output_mode_ == kSbPlayerOutputModeInvalid) {
+    PlayerErrorCB(kSbPlayerInvalid, this, kSbPlayerErrorOutputMode,
```

Comment:
Let's use `kSbPlayerErrorDecode` here.  We usually avoid modifying Starboard interface after it's frozen.  Hopefully `kSbPlayerErrorDecode` will be correctly mapped to a MediaError.

Reply:
Fixed.